ctk/common/src/main/kotlin/org/codice/compliance/utils/RestAssuredExtensions.kt

```diff
@@ -25,20 +26,38 @@ import org.codice.compliance.verification.binding.PostBindingVerifier
 import org.codice.compliance.verification.binding.RedirectBindingVerifier
 import org.codice.security.saml.SamlProtocol
 
+/** Response extension functions **/
+private const val BINDING_UNSUPPORTED_MESSAGE = "Binding is not currently supported."
+
 fun Response.determineBinding(): SamlProtocol.Binding {
     return with(this) {
         when {
             isPostBinding() -> SamlProtocol.Binding.HTTP_POST
             isRedirectBinding() -> SamlProtocol.Binding.HTTP_REDIRECT
-            else -> throw UnsupportedOperationException("Binding is not currently supported.")
+            else -> throw UnsupportedOperationException(BINDING_UNSUPPORTED_MESSAGE)
         }
     }
 }
+
 fun Response.getBindingVerifier(): BindingVerifier {
     return when (this.determineBinding()) {
         SamlProtocol.Binding.HTTP_REDIRECT -> RedirectBindingVerifier(this)
         SamlProtocol.Binding.HTTP_POST -> PostBindingVerifier(this)
-        else -> throw UnsupportedOperationException("Binding is not currently supported.")
+        else -> throw UnsupportedOperationException(BINDING_UNSUPPORTED_MESSAGE)
+    }
+}
+
+fun Response.getLocation(): String? {
```

Comment:
I don't liked "destination" because that's the name of a SAML element. That's why I went with "location". Maybe something like `getReceivingUrl`, `getRecipientUrl`, `getReceivingLocation`, etc?

Reply:
`getIntendedLocation`?